fun/lms/views.py

```diff
@@ -0,0 +1,24 @@
+import re
+from django.shortcuts import redirect
+from django.http import HttpResponse
+
+
+def richie(request):
+    RICHIE_ROUTE_REGEXP = r"^/richie/(?P<target>.*)"
+    # TODO use a settings to definie PLATFORM_RICHIE_URL
```

Comment:
To use a setting, you can can call settings.RICHIE_REDIRECT_URL and in openedx-docker you need to add the setting to docker_run_production.py to make it configurable...